### PR TITLE
fix(plugin-conversation): decrypt inbound payloads before normalizing

### LINKED ISSUE #326: helper-html is not stripping tags from post activity content fields

### PR DESCRIPTION
**Pull request by tran2**
Fixes #326

### CODE REVIEW

File: packages/plugin-conversation/test/integration/spec/verbs.js [hunk 1]
```diff
@@ -261,6 +262,72 @@ describe(`plugin-conversation`, function() {
...
+            outboundMessage: `<div><b>HELLO</b><span> it's me</span></div>`,
+            outboundFileredMessage: `<div>HELLO<span> it's me</span></div>`,
+            inboundMessage: `HELLO<span> it's me</span>`
+          }
+        ].forEach((def) => {
+          it(def.it, () => {
+            // use same rules for user 1 so that we know the message is altered
+            // as desired
+            spark.config.conversation.allowedOutboundTags = def.allowedOutboundTags;
+            spark.config.conversation.allowedInboundTags = def.allowedOutboundTags;
```
Comment (adamweeks) [inline packages/plugin-conversation/test/integration/spec/verbs.js]: Should this be: 
`spark.config.conversation.allowedInboundTags = def.allowedInboundTags;`?

File: packages/plugin-conversation/test/integration/spec/verbs.js [hunk 2]
```diff
@@ -261,6 +262,72 @@ describe(`plugin-conversation`, function() {
           assert.lengthOf(activity.object.mentions.items, 1);
           assert.equal(activity.object.mentions.items[0].id, mccoy.id);
         }));
+
+      skipInNode(describe)(`when there is html tags in rich messages`, () => {
```
Comment (ianwremmel) [inline packages/plugin-conversation/test/integration/spec/verbs.js]: "when there are"

Also, please add a comment along the lines of "disabled until helper-html has node support"

File: packages/plugin-conversation/test/integration/spec/verbs.js (hunk 1 above)
Comment (ianwremmel) [inline packages/plugin-conversation/test/integration/spec/verbs.js]: I think I convinced myself that this makes sense when I read it last night, but I'm struggling a little with it now. @tran2, can you add some more comments to explain why lines 312 and 313 work?

Comment (tran2) [inline packages/plugin-conversation/test/integration/spec/verbs.js]: I've added the comments.
What I'm thinking is that if we let allowedInboundTags to allow all the tags (kind of like disable inbound filter) then we know our messages are filtered by the outbound rules. But this is assuming inbound filter is doing what it's supposed to do.
I wonder if I should split the tests into two:
- outbound/inbound for spock
- inbound for mccoy (to test with another user)

**Pull request merged**